README.md

```diff
@@ -22,34 +18,56 @@ Here's how to set up `phylum-ci` for local development.
     git clone git@github.com:phylum-dev/phylum-ci.git
     ```
 
-2. Ensure [poetry](https://python-poetry.org/docs/) is installed
-3. Install dependencies with `poetry`, which will automatically create a virtual environment:
+2. Ensure all supported Python versions are installed locally
+   1. The strategy is to support all released minor versions of Python that are not end-of-life yet
+   2. The current list is 3.7, 3.8, 3.9, and 3.10, but the Python Developer's Guide can be referenced for the [status of active Python releases](https://devguide.python.org/#status-of-python-branches)
+   3. It is recommended to use [`pyenv`](https://github.com/pyenv/pyenv) to manage multiple Python installations
+
+    ```sh
+    # `pyenv install --list` to get available versions and usually install the latest patch version
+    pyenv install 3.7.13
+    pyenv install 3.8.13
+    pyenv install 3.9.11
+    pyenv install 3.10.3
+    pyenv rehash
+    # ensure all environments are available globally (helps tox to find them)
+    pyenv global 3.10.3 3.9.11 3.8.13 3.7.13
+    ```
+
+3. Ensure [poetry](https://python-poetry.org/docs/) is installed
+4. Install dependencies with `poetry`, which will automatically create a virtual environment:
 
     ```sh
     cd phylum-ci
     poetry install
     ```
 
-4. Create a branch for local development:
+5. Create a branch for local development:
 
     ```sh
     git checkout -b <name-of-your-branch>
     ```
 
     Now you can make your changes locally.
 
-5. When you're done making changes, check that your changes pass the tests:
+6. If new dependencies are added, ensure the `poetry.lock` file is updated (and committed):
+
+    ```sh
+    poetry lock
+    ```
+
+7. When you're done making changes, check that your changes pass the tests:
 
     ```sh
-    poetry run pytest
+    poetry run tox
```

Comment:
In line with my comment above about `poetry run tox` in the CI pipeline, I think we can and should encourage developers to install `tox` globally and run it directly here

```suggestion
    tox
```

_Note:_ We'd need to add an item above about installing `tox` (probably with `pipx install tox`)

Reply:
Yeah...it wouldn't take much convincing to adopt this model.

The thought about why to prefer `poetry run tox` over simply `tox` is that it would allow for that much more control over consistent, repeatable behavior by locking the version of `tox` (and it's dependencies) along with all the other `dev-dependencies` specified in the `pyproject.toml` file. The addition of groups with the release of poetry 1.2 should make all of this much better.